installation_guide.md

```diff
@@ -0,0 +1,72 @@
+## Installation and Requirements
+
+### Windows
+
+1. Install [Podman Desktop](https://podman-desktop.io/downloads/Windows)
+2. Install [WSL 2 from the Microsoft Store](https://apps.microsoft.com/store/detail/windows-subsystem-for-linux/9P9TQF7MRM4R)
+3. Install python and pip if not already present. Then install the `podman-compose` package using the following command:
+
+   ```
```

Comment:
I like to give a suggestion on what language it is to get some code colors (works with pip, sudo, etc...)

Consider changing all of the code chunks to have the respective code highlights (bash, R, dockerfile)

![image](https://github.com/Appsilon/experimental-fda-submission-4-podman/assets/211358/d62cf58f-0e67-4c9d-9f15-f333f95c8ef5)


```suggestion
   ```bash
```